scripts/tasks/sandbox-parts.ts

```diff
@@ -787,6 +787,17 @@ export const extendMain: Task['run'] = async ({ template, sandboxDir, key }, { d
   await writeConfig(mainConfig);
 };
 
+export const extendPreview: Task['run'] = async ({ template, sandboxDir }) => {
+  logger.log('📝 Extending preview.js');
+  const previewConfig = await readConfig({ cwd: sandboxDir, fileName: 'preview' });
+
+  if (template.expected.builder.includes('vite')) {
+    previewConfig.setFieldValue(['tags'], ['vitest']);
```

Comment:
I realize this is unrelated to the PR how does the `vitest` tag differ from the `test` tag?

Reply:
We use both the test-runner and vitest runner. There are things that we can't test in vitest that we can in the test-runner (e.g. channels). So we decided to disable tests via `!test` for the test-runner and `!vitest` for vitest. Normal users will likely not be in this scenario so `!test` should be enough for them